helm/monitoring/loki-values.yaml

```
@@ -14,4 +10,4 @@ promtail:
       - url: http://loki.monitoring.svc:3100/loki/api/v1/push
 
 grafana:
-  enabled: false
\ No newline at end of file
+  enabled: false                   # 이미 설치된 경우 false로 유지
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)